benchmarks/startup/scenario.py

```diff
@@ -0,0 +1,38 @@
+import os
+import subprocess
+import sys
+
+import bm
+
+
+class Startup(bm.Scenario):
+    telemetry_enabled = bm.var_bool()
+    runtime_metrics_enabled = bm.var_bool()
+    use_ddtrace_run = bm.var_bool()
+    with_trace = bm.var_bool()
+
+    def run(self):
+        code = "from ddtrace import tracer"
+        if self.with_trace:
+            # queue and encode trace but don't send to the agent
+            # requests to the agent will raise an exception in a benchmark test
+            code += """
+import mock
+tracer._writer._send_payload = mock.Mock()
+with tracer.trace('test-x', service='bench-test'):
+    pass
+"""
+
+        cmd = [sys.executable, "-c", code]
```

Comment:
Note for the following command `scripts/perf-run-scenario startup ddtrace==1.0.1 ddtrace==0.60.1 ./artifacts/` the startup benchmark raises the following error when running benchmarks for ddtrace v0.60.1:

```
Traceback (most recent call last):
  File "/app/scenario.py", line 7, in <module>
    class Startup(bm.Scenario):
  File "/app/bm/_scenario.py", line 54, in __init__
    _register(cls)
  File "/app/bm/_scenario.py", line 41, in _register
    runner.bench_time_func(scenario.scenario_name, scenario._pyperf)
  File "/app/.venv_ddtrace_v2/lib/python3.9/site-packages/pyperf/_runner.py", line 462, in bench_time_func
    return self._main(task)
  File "/app/.venv_ddtrace_v2/lib/python3.9/site-packages/pyperf/_runner.py", line 432, in _main
    bench = self._manager()
  File "/app/.venv_ddtrace_v2/lib/python3.9/site-packages/pyperf/_runner.py", line 618, in _manager
    self._display_result(bench)
  File "/app/.venv_ddtrace_v2/lib/python3.9/site-packages/pyperf/_runner.py", line 601, in _display_result
    pyperf.add_runs(args.append, bench)
  File "/app/.venv_ddtrace_v2/lib/python3.9/site-packages/pyperf/_bench.py", line 864, in add_runs
    suite.add_runs(result)
  File "/app/.venv_ddtrace_v2/lib/python3.9/site-packages/pyperf/_bench.py", line 667, in add_runs
    self._add_benchmark_runs(result)
  File "/app/.venv_ddtrace_v2/lib/python3.9/site-packages/pyperf/_bench.py", line 663, in _add_benchmark_runs
    existing.add_runs(benchmark)
  File "/app/.venv_ddtrace_v2/lib/python3.9/site-packages/pyperf/_bench.py", line 586, in add_runs
    self.add_run(run)
  File "/app/.venv_ddtrace_v2/lib/python3.9/site-packages/pyperf/_bench.py", line 444, in add_run
    raise ValueError("incompatible benchmark, metadata %s is "
ValueError: incompatible benchmark, metadata python_executable is different: current=/app/.venv_ddtrace_v1/bin/python, run=/app/.venv_ddtrace_v2/bin/python
```